website/client/src/components/tasks/spells.vue

```diff
@@ -76,138 +62,199 @@
-        color: #686274;
-      }
+    .mana{
+      background-color: rgba(26, 24, 29, 0.5);
     }
+  }
+
+  .details {
+    text-align: center;
+    padding-top: 0.5em;
+    min-height: 4em;
+    width: 3.6em;
 
     .img {
-      margin: 0 auto;
+      display: inline-block;
     }
 
-    .mana-text {
-      margin-bottom: .2em;
-      padding-top: 1.1em;
+    span {
+      display: inline-block;
+      width: 50%;
+      padding-bottom: 0.7em;
+      vertical-align: bottom;
+    }
+  }
 
-      div {
-        display: inline-block;
-        vertical-align: bottom;
-      }
+  .img {
+    margin: 0 auto;
+  }
 
-      .svg-icon {
-        width: 16px;
-        height: 16px;
-        margin-right: .2em;
-      }
+  .mana-text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    div {
+      display: inline-block;
+      vertical-align: bottom;
     }
 
-    .mana {
-      padding: .2em;
-      background-color: rgba(70, 167, 217, 0.24);
-      color: #2995cd;
-      font-weight: bold;
-      text-align: center;
+    .svg-icon {
+      width: 16px;
+      height: 16px;
+      margin-right: 0.2em;
     }
   }
 
-  .dragInfo {
-    position: absolute;
-    left: -500px;
-    z-index: 1080;
-
-    .spell {
-      border-radius: 1000px;
-      min-width: 224px;
-      height: 52px;
-      font-size: 12px;
-      padding-left: .5em;
-
-      .title {
-        font-weight: bold;
-        margin-bottom: .2em;
+  .mana {
+    background-color: rgba(70, 167, 217, 0.24);
+    color: #2995cd;
+    font-weight: bold;
+    text-align: center;
+    min-height: 2em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+  }
+}
+
+.dragInfo {
+  position: absolute;
+  left: -500px;
+  z-index: 1080;
+  color: #e1e0e3;
+  border: none;
+  background-color: transparent;
+  box-shadow: transparent;
+
+  .spell {
+    border-radius: 4px;
+    width: 93px;
+    height: 7em;
+    padding-bottom: 0;
+    margin-bottom: 0;
+    background-color: transparent;
+    border: none;
+    box-shadow: transparent;
+    .spell-border{
+      clip-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
+      width: 62px;
+      height: 62px;
+      background-color: #46a7d9;
+      margin-left: 1.1em;
+      margin-bottom: 0.5em;
+
+      .mana{
+        width: 60px;
+        height: 60px;
+        clip-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
+        background-color: #f9f9f9;
+        border-radius: 4px;
+        margin-top: 1px;
+        margin-bottom: 1px;
+        margin-left: 1px;
+        margin-right: 1px;
+
+        .img{
+          background-color: #f9f9f9;
+          clip-path: polygon(50% 0%, 100% 50%, 50% 100%, 0% 50%);
+        }
       }
     }
+    .details{
+      width: 100%;
+      height: 1em;
+      border-radius: 4px;
+      background-color: rgba(52, 49, 58, 0.96);
 
-    .mana {
-      border-radius: 0 1000px 1000px 0;
+      .notes {
+        font-family: Roboto;
```

Comment:
font-family, font-weight, font-stretch, font-style, letter-spacing properties aren't necessary. Zeplin includes them but these are default values that don't have to be specified